include/pmacc/identifier/alias.hpp

```diff
@@ -90,7 +52,19 @@ identifier(pmacc_isAlias);
  * get type which is represented by the alias
  *      typedef typename traits::Resolve<name>::type resolved_type;
  */
-#define alias(name) PMACC_alias(name,__COUNTER__)
+#define alias(name)                                                            \
+        template<                                                              \
+            typename T_Type = pmacc::pmacc_void,                               \
+            typename T_IsAlias = pmacc::pmacc_isAlias                          \
+        >                                                                      \
+        struct name                                                            \
+        {                                                                      \
+            static std::string getName()                                       \
+            {                                                                  \
+                return std::string(#name);                                     \
```

Comment:
why `#`?

Reply:
It is to get stringified version of the name, in quotes.

Reply:
@sbastrakov Thnaks for the explanation

Reply:
That allowed me to google: is a pre-compiler command: https://gcc.gnu.org/onlinedocs/gcc-4.8.5/cpp/Stringification.html